index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Mobile</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="desktop-nav">
+        <div class="menu-icon">
+            <img src="images/ic_menu.png" alt="">
+        </div>
+        <div class="logo-mobile">
+            <img src="images/image_geometry_m_index.png" alt="">
+        </div>
+        <ul class="menu-items">
+            <li><a href="index.html">Hello</a></li>
+            <li><a href="Portfolio">Portfolio</a></li>
+            <li><a href="About">About</a></li>
+            <li><a href="Contact">Contact</a></li>
+        </ul>
+    </nav>
+    <header class="webcontainer">
```

Comment:
- [ ] This section should be enclosed in section tag

Reply:
Semantic also not necessary at this point. Of course, it's a good way to fix this concept for me, but the primary is to prepare the skeleton.

Reply:
I am sorry but that is a requirement